specification/serialconsole/resource-manager/Microsoft.SerialConsole/preview/2018-07-01-preview/serialconsole.json

```diff
@@ -0,0 +1,272 @@
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/default"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK - Returns a JSON object",
+                        "schema": {
+                            "$ref": "#/definitions/GetDisabledResult"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/DeploymentValidateResult"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Get whether or not Serial Console is disabled": {
+                        "$ref": "./examples/GetConsoleDisabledExample.json"
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.SerialConsole/providers/consoleServices/{default}/enableConsole": {
+            "post": {
+                "tags": [
+                    "EnableConsole"
+                ],
+                "operationId": "Console_enableconsole",
+                "description": "Enables Serial Console for a VM",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/default"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK - Serial Console disabled",
+                        "schema": {
+                            "$ref": "#/definitions/SetDisabledResult"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/DeploymentValidateResult"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Enable console successfully": {
+                        "$ref": "./examples/EnableConsoleExample.json"
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.SerialConsole/providers/consoleServices/{default}/disableConsole": {
+            "post": {
+                "tags": [
+                    "DisableConsole"
+                ],
+                "operationId": "Console_disableconsole",
```

Comment:
For better SDK user experience, I would reccomend to use `CamelCase` for operation id. For example `Console_DisableConsole`.